main.go

```diff
@@ -7,17 +7,23 @@ import (
 
 	"github.com/ndv6/tsaving/api"
 	"github.com/ndv6/tsaving/database"
+	"github.com/ndv6/tsaving/helpers"
 )
 
 func main() {
-	db, err := database.GetDatabaseConnection("host=127.0.0.1 port=5432 user=postgres password=password dbname=db_tsaving sslmode=disable")
+	config, err := helpers.LoadConfig("configs/configs.json")
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	fmt.Println("Server is now accepting request from port 8000")
-	err = http.ListenAndServe("127.0.0.1:8000", api.Router(db))
+	db, err := database.GetDatabaseConnection(config.DbCfg)
 	if err != nil {
-		log.Fatal("Can not listen to port 8000", err)
+		log.Fatal(err)
```

Comment:
I think this part will never run because the previous log.Fatal() will stop the program